sdk/python/lib/pulumi/resource.py

```diff
@@ -865,7 +873,7 @@ def export(name: str, value: Any):
     :param str name: The name to assign to this output.
     :param Any value: The value of this output.
     """
-    stack = get_root_resource()
+    stack = cast(Stack, get_root_resource()) # pylint: disable=used-before-assignment
```

Comment:
We should probably actually fix this - either make `get_root_resource` return a `Stack` type, or have this code do an `isinstance` and fail if it's not a stack?

Also - is the pylint error about `Stack`?  If so - we should just make it `'Stack'`.

Reply:
Added stack to known_types and made this change.